src/app/public/modules/country-field/fixtures/country-field-no-form.component.fixture.html

```diff
@@ -0,0 +1,9 @@
+<div>
+  <sky-country-field
+    [disabled]="isDisabled ? true : undefined"
+    [defaultCountry]="defaultCountry"
+    [attr.required]="isRequired ? true : undefined"
```

Comment:
Check your alphabetical ordering of properties here.

Reply:
Done